doc/1.0/tutorial.adoc

```diff
@@ -233,7 +233,13 @@ var realTypeof = x === null ? "null" :
                  typeof x === "undefined" ? "undefined" : typeof x);
 ----
 
-Since delimiters nest syntax in Sweet, we need a way to get at what is inside them. Syntax objects have a `inner` method to do just that; calling `inner` on a delimiter will return an iterator into the syntax inside the delimiter.
+Since delimiters nest syntax in Sweet, we need a way to get at the nested syntax. The `contextify` method on the macro context provides this functionality. Calling `contextify` with a delimiter will wrap the syntax inside the delimiter in a new macro context object that you can iterate over.
+
+Utility functions are available at `'sweet.js/helpers'` for import to help inspect what kind of syntax you are processing. Note the `for syntax` on the import declaration, which is required to make the imports available inside of a macro definition. Modules and macros are described in the <<_sweet_modules, modules>> section of this document.
```

Comment:
The wording here is a little awkward. Maybe:

"Utility functions are available for import from `'sweet.js/helpers'` to [inspect the | help determine what kind of] syntax you are processing."

Incidentally, this doesn't work in the browser. I don't think `helpers` has been passed to the browser store.